packages/table/stories/CheckboxTable.stories.tsx

```diff
@@ -151,4 +151,43 @@ storiesOf("Table/CheckboxTable", module)
       />
     </CheckboxTableHelper>
   ))
+  .addWithInfo("w/ disabled row", () => (
+    <CheckboxTableHelper
+      data={items}
+      disabledRows={{ [items[0].name]: true }}
```

Comment:
This seems like an unnecessarily complex data structure to me. Can we do something like
`disabledRows = [items[0].name, items[1].name]`?

Reply:
I wanted to be consistent with the data structure for `selectedRows`. Or do you think it's so complex that it's not worth being consistent with `selectedRows`?

Reply:
In that case I prefer consistency then.